packages/e2e/scripts/playwright/axe-clean.spec.ts

```diff
@@ -47,7 +47,7 @@ let page: Page
 
 let allViolations: Result[] = []
```

Comment:
@rezrah I wanted to improve this playwright test so bad, but I realized few things:
1. This variable never aggregates data, it get's reset for every test - tests don't share states in playwright. (I wonder if the browser is shared though, could try to share the state through browsers localstorage) The only way to share the state is to write it to a file - but that poses another issue 👇 
2. `beforeAll` and `afterAll` get triggered on every test fail - maybe it's a bug, maybe I'm setting things in a bad way, but what I'm reading it's how it works? `beforeAll` and `afterAll` are currently in the `for loop`, and I don't think that's good and performant. This makes these hooks run for every story...

I tried sharing state through a file, but then because `beforeAll` works unexpectedly I will need to have an additional two scripts that get executed before and after running of playwright to set and clean the stage for the state file.

I think this is better suited for another PR. I think I made a big progress.